SConstruct

```diff
@@ -674,7 +674,7 @@ for variant_path in variant_paths:
 
     if env['GCC']:
         gcc_min_version = "11"
-        gcc_max_version = "14"
+        gcc_max_version = "14.2"
```

Comment:
Ah, I get it now. For future ref: the comparison function fills missing major/minor/hotfix values with zero. So 14 would be translated to 14.0.0. likewise 14.1 would be translated to 14.1.0.